adguard-mail.com/eula.html.md

```diff
@@ -0,0 +1,139 @@
+
+1.4. AdGuard Mail that AdGuard Software Limited may provide in accordance with clause 1.2 on a free basis may have a number of limitations, such as restrictions on the number of email aliases you can create and the number of recipients you can forward emails to. Information on these and other limitations is provided on the Website.
+
+1.6. To use AdGuard Mail, you must install the application or visit the Website and sign in using your email address or sign up by creating a new account. You must provide a valid email address when signing up. You may also sign in with an email and password you have already registered on the Website or other Rightholder’s websites, such as adguard.com, previously.
+
+1.7. We may modify or discontinue AdGuard Mail free services and benefits without notice at any time at our sole discretion.
+
+1.8 Continued use of AdGuard Mail after any modifications to this EULA have been made will constitute your acceptance of those changes.
+
+## 2. Restrictions
+
+2.1. You may not make or distribute copies of the Software or electronically transfer the Software from one device to another or over a network.
+
+2.2. You may not alter, merge, modify, or adapt the Software, or decompile, reverse engineer, disassemble, or otherwise reduce the Software to a human-perceivable form, except to the extent permitted by applicable law notwithstanding this restriction or to the extent otherwise expressly permitted by the Rightholder.
+
+2.3. You may not sell, resell, rent, lease, or sublicense the Software. Only authorized Partners of Rightholder are permitted to resell or sublicense the Software.
+
+2.4. You may not modify the Software or create derivative works based upon the Software.
+
+2.5. In the event that you fail to comply with this EULA, Rightholder may terminate all licenses to the Software, and you must destroy all copies of the Software.
+
+2.6. Unless otherwise provided herein, you shall not when using the beta version of the Software
+
+(A) use the beta version of the Software for a purpose other than the sole purpose of testing the Software,
+
+(B) use the beta version of the Software if you do not participate in the Beta testing program of the Rightholder or if you violate the rules of the Beta testing program,
+
+(C) distribute, share, send, or make the beta version of the Software otherwise available to third parties without prior written consent of the Rightholder, or
+
+(D) breach the confidentiality of the beta version of the Software, its functions, features, characteristics, design, user interface, documentation, or other related materials by publishing, sharing, sending, disclosing, or otherwise making available to third parties without prior written consent of the Rightholder.
```

Comment:
```suggestion
(D) breach the confidentiality of the beta version of the Software, its functions, features, characteristics, design, user interface, documentation, or other related materials by publishing, sharing, sending, disclosing, or otherwise making them available to third parties without prior written consent of the Rightholder.
```